packages/realm_dart/lib/src/handles/native/mutable_subscription_set_handle.dart

```diff
@@ -86,5 +93,11 @@ class MutableSubscriptionSetHandle extends SubscriptionSetHandle {
     });
   }
 
+  @override
   void clear() => realmLib.realm_sync_subscription_set_clear(_mutablePointer).raiseLastErrorIfFalse();
+
+  @override
+  // Workaround for weird compiler bug
+  // ignore: unnecessary_overrides
+  SubscriptionHandle? findByResults(covariant ResultsHandle results) => super.findByResults(results);
```

Comment:
Can we add a bit more details for the weird compiler bug? Just so that we don't have to guess what that meant 6-12 months from now.